lab04/README.md

```diff
@@ -2,70 +2,46 @@
 
 ## Objectives
 
-- Understand how you can adapt terraform in projects with existing infrastructure
-- Import existing resources to bind them with Terraform `state`
-- Modify `state` by removing and importing objects
+- Import existing resources to bind them into Terraform `state`
 - Inspect `state` 
 - Add rule to ignore certain changes in objects
 
 ## Terraform state
 
-Terraform keeps information about effect of its work in `state`. 
-In this scenario you will create resources manually and then map them with configuration files.
-After that you will remove resource from `state` and apply configuration.
-
-Key points:
-- effectively this is text file which helds information about objects managed by Terraform
-- `state` is what Terraform _believes_ is out there, it is being used to compare expected vs existing state
-- terraform allows to bind existing resource with state using import command
+Terraform keeps information about effect of its work in file `terraform.tfstate` often referred as`state` or `state file`. 
+In this scenario we will create resources manually and then import them into Terraform state.
+After that we will remove resource from `state` and apply configuration.
 
 ### Importing resources
    
-1. Create sample resources using bash script
-    - inspect [/scripts/createRGSA.sh](./scripts/createRGSA.sh) and put your initials in relevant places
-    - run script
-       ```bash
-       cd scripts
-       chmod +x createRGSA.sh
-       ./createRGSA.sh
-       ```
-    - note `RGNAME` and `SANAME` values on the side
+1. Inspect script [createRGSA.sh](./scripts/createRGSA.sh), put your initials in relevant places and run it. 
+Note `RG_NAME` and `SA_NAME` values on the side.
   
-2. Compare terraform config versus exising state
-    - initialize terraform
-    - run `terraform plan` in infra directory and pass Resource Group name and Storage Account name values as parameters to execution
-      ```bash
-      cd infra
-      terraform plan -var="rg_name=<RGNAME>" -var="sa_name=<SANAME>"
-      ```
-      Notice that terraform wants to create all resources.
+2. Run `terraform plan` and compare execution plan with exising infrastructure on the azure portal. 
```

Comment:
Why remove variable input? Maybe add a note about changing defaults in variables.tf? 
```suggestion
2. Run `terraform plan` and compare the execution plan with existing infrastructure on the Azure portal. 
```

Reply:
added note about variable imputs